client/components/common/ManageRequiredReportsDisclosure/CreateRequiredReportForm.jsx

```diff
@@ -0,0 +1,208 @@
+  background-repeat: no-repeat;
+  background-position: right 0.75rem center;
+  background-size: 16px 12px;
+  float: left;
+  margin-top: 2px;
+  white-space: nowrap;
+  background-color: ${props =>
+    props.phaseLabel === 'Select a Phase'
+      ? '#fff'
+      : props.phaseLabel === 'Candidate'
+      ? '#ff6c00'
+      : props.phaseLabel === 'Recommended'
+      ? '#8441de'
+      : 'black'};
+  border-radius: 14px;
+  padding: 2px 32px 2px 14px;
+  text-align: left;
+  width: 100%;
+  font-size: 1rem;
+  font-weight: 400;
+  color: ${props => (props.phaseLabel === 'Select a Phase' ? 'black' : '#fff')};
+`;
+
+const CustomMenu = React.forwardRef(({ children, className }, ref) => {
+  const value = '';
+
+  return (
+    <div ref={ref} className={className}>
+      <ul>
+        {React.Children.toArray(children).filter(
+          child =>
+            !value || child.props.children.toLowerCase().startsWith(value)
+        )}
+      </ul>
+    </div>
+  );
+});
+
+// You can learn everything about this component here: https://react-bootstrap.netlify.app/docs/components/dropdowns#custom-dropdown-components
+const CustomToggle = React.forwardRef(({ children, onClick }, ref) => (
+  <CustomToggleButton
+    ref={ref}
+    onClick={e => {
+      e.preventDefault();
+      onClick(e);
+    }}
+  >
+    <CustomToggleP>
+      <CustomToggleSpan phaseLabel={children}>{children}</CustomToggleSpan>
+    </CustomToggleP>
+  </CustomToggleButton>
+));
+
+const FormGroup = ({ label, children }) => (
+  <Form.Group className="form-group">
+    <Form.Label className="disclosure-form-label">{label}</Form.Label>
+    {children}
+  </Form.Group>
+);
+
+export const CreateRequiredReportForm = ({ ats, handleCreate }) => {
+  const [formState, setFormState] = useState({
+    phase: '',
+    at: '',
+    browser: ''
+  });
+
+  const handleInputChange = (field, value) => {
+    setFormState(prev => ({ ...prev, [field]: value }));
+  };
+
+  const handleSubmit = async () => {
+    await handleCreate({
+      atId: formState.at,
+      browserId: formState.browser,
+      phase: formState.phase.toUpperCase()
+    });
+    setFormState({ phase: '', at: '', browser: '' });
+  };
+
+  return (
+    <div className="disclosure-row-controls">
+      <FormGroup label="Phase">
+        <Dropdown>
+          <Dropdown.Toggle as={CustomToggle} id="dropdown-custom-components">
+            {formState.phase || 'Select a Phase'}
+          </Dropdown.Toggle>
+          <Dropdown.Menu className="drop-down-div" as={CustomMenu}>
+            {['Candidate', 'Recommended'].map(phase => (
+              <Dropdown.Item
+                key={phase}
+                className="phase-option"
+                onClick={() => handleInputChange('phase', phase)}
+              >
+                {phase}
+              </Dropdown.Item>
+            ))}
+          </Dropdown.Menu>
```

Comment:
This custom menu might be a bit much for just this as I'm not seeing a major advantage to highlighting the selected option's color. It also feels a bit out of place given this and the connected disclosures' options. This feels like a case where simplicity may be better, with a native `<select>`.

Reply:
Addressed in 43df566